.github/workflows/pr-check-file-encodings.yaml

```diff
@@ -0,0 +1,21 @@
+name: check file encodings in PR
```

Comment:
This workflow seems broadly useful. Would it make sense to add it to [r2dii.usethis/inst/templates](https://github.com/2DegreesInvesting/r2dii.usethis/tree/master/inst/templates) so we can add it wherever is needed with something like this?:

    usethis::use_template("pr-check-file-encondings.yaml", package = "r2dii.usethis")


Beyond the convenient way to add it, there is also the benefit of defining a central place to maintain it.  @cjyetman, what the current home of this workflow?

Reply:
https://github.com/2DegreesInvesting/resources/issues/53
🤣

Reply:
maybe this? https://docs.github.com/en/actions/learn-github-actions/sharing-workflows-with-your-organization#creating-a-workflow-template

Reply:
I started something related to this here https://github.com/2DegreesInvesting/.github/pull/6